src/Autodesk.Forge.DesignAutomation/ServiceCollectionExtensions.cs

```diff
@@ -36,8 +36,7 @@ public static partial class ServiceCollectionExtensions
         public static void AddDesignAutomationFactory(this IServiceCollection services, IConfiguration configuration, Action<IHttpClientBuilder> configureBuilder = null)
         {
             IHttpClientBuilder builder;
-            var forgeConfiguration = configuration.GetSection("Forge").Get<ForgeConfiguration>();
-            foreach (var agent in forgeConfiguration.Agents)
+            foreach (var agent in configuration.GetSection("Forge:Agents").GetChildren())
             {
```

Comment:
`forgeConfiguration.Agents` is null, foreach loop throw NullRefException. I found it when running on appsettings without  `Agents` defined. 
revert to the old code which works fine